build.ts

```diff
@@ -32,11 +31,18 @@ const accents = [
 
 async function makeThemeObject(
   name: string,
-  accent: keyof Labels<Color, AlphaColor>,
-  palette: Labels<Color, AlphaColor>
+  accent: string,
```

Comment:
```suggestion
  accent: AccentName,
```
You can use the `AccentName` type from `@catppuccin/palette`.